integrations/langchain4j/providers/open-ai/src/test/java/io/helidon/integrations/langchain4j/openai/ImageModelConfigTest.java

```diff
@@ -65,13 +64,12 @@ void testDefaultRoot() {
         assertThat(config.logRequests().get(), is(true));
         assertThat(config.logResponses().isPresent(), is(true));
         assertThat(config.logResponses().get(), is(true));
-        assertThat(config.withPersisting().isPresent(), is(true));
-        assertThat(config.withPersisting().get(), is(true));
-        assertThat(config.persistTo().isPresent(), is(true));
-        assertThat(config.persistTo().get(), is(Paths.get("temp")));
+//        assertThat(config.withPersisting().isPresent(), is(true));
```

Comment:
Do not comment our lines - if these are no longer relevant, delete them. If they are relevant, fix it or create a follow up issue.